selfdrive/ui/qt/widgets/cameraview.cc

```diff
@@ -142,30 +146,36 @@ void CameraViewWidget::initializeGL() {
   glBufferData(GL_ELEMENT_ARRAY_BUFFER, sizeof(frame_indicies), frame_indicies, GL_STATIC_DRAW);
   glBindBuffer(GL_ARRAY_BUFFER, 0);
   glBindVertexArray(0);
-
-  vipc_client = std::make_unique<VisionIpcClient>("camerad", stream_type, true);
 }
 
 void CameraViewWidget::showEvent(QShowEvent *event) {
-  timer->start(0);
+  VisionStreamType type = stream_type;
+  if (type != VISION_STREAM_RGB_FRONT) {
+    bool wide_cam = Hardware::TICI() && Params().getBool("EnableWideCamera");
+    type = wide_cam ?  VISION_STREAM_RGB_WIDE : VISION_STREAM_RGB_BACK;
+  }
```

Comment:
the CameraViewWidget shouldn't care about this. it should just show the vision stream it's told to